examples/LexActivator.h

```diff
@@ -1114,7 +1219,7 @@ LEXACTIVATOR_API int LA_CC GenerateOfflineDeactivationRequest(CSTRTYPE filePath)
 
     RETURN CODES: LA_OK, LA_EXPIRED, LA_SUSPENDED, LA_GRACE_PERIOD_OVER, LA_FAIL,
     LA_E_PRODUCT_ID, LA_E_LICENSE_KEY, LA_E_TIME, LA_E_TIME_MODIFIED, LA_E_MACHINE_FINGERPRINT,
-    LA_E_RELEASE_VERSION_NOT_ALLOWED, LA_E_OS_USER
+    LA_E_RELEASE_VERSION_NOT_ALLOWED, LA_E_OS_USER, LA_E_FEATURE_ENTITLEMENTS_INVALID

```

Comment:
@ahmad-kemsan what is this error code

Reply:
@adnan-kamili In case the feature entitlement token is invalid (maybe does not belong to that license or activation or maybe tampered. JWT can be valid but the activationId may not match), we return this error. We even allow server sync on so that sync can make sure the correct feature entitlement is placed.

Reply:
ok